Packs/ThreatZone/Integrations/ThreatZone/ThreatZone.py

```diff
@@ -17,6 +17,10 @@
 """ CLIENT CLASS """
 
 
+def bool_converter(bool_as_str: str) -> bool:
```

Comment:
Note that we have the `argToBoolean` function in `CommonServerPython` so you can use it.